api-documentation/complete-api-documentation.md

```diff
@@ -2,16 +2,16 @@
 
 ## Story
 
-As a API consumer
-I want to be able to get an extensive description of the API's data structures and capabilities
-so that I can use API testing tools, make some client-side pre-computations or create a documentation out of it.
+> As a API consumer<br>
```

Comment:
Is there anything we can do about these HTML tags?

Reply:
Wat do you want to do about them? 🤣